docs/accessibility/design.md

```diff
@@ -56,25 +51,29 @@ Small foreground text (non-bold text under 24px and bold text under 19px) must h
   <img src="{{ '/assets/color/contrast-text.png' | url }}" alt="Contrast ratios for dark gray sections with black text that uses different weights and fonts">
 </uxdot-example>
 
-
 #### Links
 
-If color is the only way to distinguish between links and surrounding text (e.g., if link underlines are removed within a paragraph), the contrast ratio between the link and surrounding text must be at least 3:1.
+Inline text links (e.g., those within paragraphs and list items in the main body of a page) should be underlined to make these links easy to discern for all visitors, particularly those with visual impairments or in situations that could make color differentiation difficult.
+
+In fact, you should default to underlining all text links, except for the following:
 
-Non-color cues should be used to signify when a link receives hover and focus (e.g., an underline).
+- Links within visually distinct navigation groupings, like menus.
+- CTAs and other links accompanied by visual cues (e.g., arrow symbols) to indicate their interactivity.
+
+That said, you still _can_ underline the above, at your discretion.
+
+If, for some reason, color is the only way you can distinguish links within a text block, the contrast ratio between the link and surrounding text _must_ be at least 3:1 in both visited and unvisited states. And underlines or other non-color cues _must also_ then be used to signify when the link receives hover and focus, as in the example below:
 
 <uxdot-example width-adjustment="692px" no-border>
   <img src="{{ '/assets/color/contrast-links.png' | url }}" alt="Contrast ratio of a blue link next to black text and an example of a link's darker blue, underlined hover state">
```

Comment:
I like this image, but I think we need to update it to show the underlined text for the default state. Maybe @coreyvickery or @marionnegp can re-export it?

Reply:
@marionnegp @gradymcgee Can one of you re-export? I see the image has been updated in Figma already.

Reply:
Thanks, y'all

Reply:
Thanks, everyone! The new image is in the DP, @markcaron . Feel free to re-review.

Reply:
Looks good!